docs/manual/troubleshooting.tex

```diff
@@ -1124,6 +1124,7 @@ \section{Credits and changelog\label{credits}}
 David Lazar,
 David McArthur,
 Eric Spishak,
+Google Inc. (via @wmdietlGC),
```

Comment:
Nit: please put `\` after `.` to signal to LaTeX that this period doesn't end a sentence.